packages-exp/performance-exp/src/services/perf_logger.test.ts

```diff
@@ -98,6 +96,29 @@ describe('Performance Monitoring > perf_logger', () => {
   });
 
   describe('logTrace', () => {
+    it('will not drop event before initialization finishes', async () => {
```

Comment:
How about something like "will not drop custom events sent before initialization finishes" to make it more specific?

Reply:
Done.